Cron/PendingOrdersUpdate.php

```diff
@@ -36,8 +36,9 @@ public function __construct(
 
     /**
      * @throws LocalizedException
+     * @return self
      */
-    public function execute(): static
+    public function execute()
```

Comment:
If you do want to keep the return type hint you can use the classname instead of `static`